From c016b0c796b60f33f9c1396ff830cb7838abbba6 Mon Sep 17 00:00:00 2001
From: Jason Montleon <jmontleo@redhat.com>
Date: Sun, 19 Oct 2025 12:07:56 -0400
Subject: [PATCH 1233/1234] Revert "tty: serial: atmel: make it selectable for
 ARCH_MICROCHIP"

This reverts commit 9fc0a7e0ee896b1b4e66b1ffe38efe2f4af20891.
---
 drivers/tty/serial/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/tty/serial/Kconfig b/drivers/tty/serial/Kconfig
index 748129a4fd5be..69b7cf065c1c7 100644
--- a/drivers/tty/serial/Kconfig
+++ b/drivers/tty/serial/Kconfig
@@ -128,7 +128,7 @@ config SERIAL_SB1250_DUART_CONSOLE
 config SERIAL_ATMEL
 	bool "AT91 on-chip serial port support"
 	depends on COMMON_CLK
-	depends on ARCH_MICROCHIP || COMPILE_TEST
+	depends on ARCH_AT91 || ARCH_LAN969X || COMPILE_TEST
 	select SERIAL_CORE
 	select SERIAL_MCTRL_GPIO if GPIOLIB
 	select MFD_AT91_USART
-- 
2.51.0

